spec/lib/seeds_spec.rb

```diff
@@ -0,0 +1,11 @@
+require 'rails_helper'
+
+RSpec.describe Rails::Application do
```

Comment:
Curious to get thoughts on this:

- is it a good idea to add this test?
- is this the best way of fitting it into the `spec/` directory structure?

Reply:
I think having this test is a good idea, if we had had it before we would have caught the breakages as they happened.

I'm not sure if there are conventions for fitting this test in but this seems quite readable to me!

Reply:
👍